docker/build-docker.sh

```diff
@@ -48,6 +49,77 @@ send-error()
     exit 1
 }
 
+# Prints compatibility matrix by scanning .github/workflows/containers.yml matrix
+show-matrix()
+{
+    local filter_distro="${1:-}"
+    local workflow_file="../.github/workflows/containers.yml"
```

Comment:
Nice, but you probably don't want to hardcode the path. If you run the script from the project root folder this will fail. 
`docker/build-docker.sh` rather than `./build-docker.sh`

Reply:
You could add 

```
SCRIPT_DIR=$(realpath $(dirname ${BASH_SOURCE[0]}))
cd $(dirname ${SCRIPT_DIR})
```
To the top of the script, then  change this path to:

```suggestion
    local workflow_file=".github/workflows/containers.yml"
```